README.md

```diff
@@ -4,6 +4,29 @@ The ultimate AI assistant for Platform Engineers, crafted by the Botkube Team.
 
 ![Using fuse in CLI](docs/assets/prompt.gif)
 
+## Usage
+
+After downloading the CLI, you can start experimenting right away!
+
+### Chat mode
+
+To start a session in chat mode, simply run:
+
+```shell
+fuse
+```
+
+### Inline prompting
+
+> [!TIP]
+> When using inline mode, enclose your prompt in single quotation marks (') to prevent your shell from interpreting it.
+
+Fuse CLI also supports inline mode, allowing you to execute prompts directly. Run:
+
+```shell
+fuse 'list files in the current directory and provide me better organization structure'
+```
```

Comment:
Can you add info about `-C`? That could be helpful too 👍 

Also, another section about the scenarios?